### PR TITLE
Compliance Operator 1.3.1 release notes

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.11+

Issue:

Link to docs preview (VPN required) :
[OpenShift Compliance Operator 1.3.1](https://file.rdu.redhat.com/antaylor/CO-1.3.1/security/compliance_operator/compliance-operator-release-notes.html#compliance-operator-release-notes-1-3-1)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 1]
```diff
@@ -15,6 +15,36 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
+//
+// [IMPORTANT]
+// ====
+// It is recommended to update the Compliance Operator to version 1.3.1 or later before updating your {product-title} cluster to version 4.14 or later.
+// ====
+
+[id="compliance-operator-1-3-1-new-features-and-enhancements"]
+=== New features and enhancements
+
+* You can install use the Compliance Operator in an {product-title} cluster in FIPS mode.
```
Comment (GroceryBoyJr) [inline security/compliance_operator/compliance-operator-release-notes.adoc:37]:
```suggestion
* You can install and use the Compliance Operator in an {product-title} cluster in FIPS mode.
```

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 2]
```diff
@@ -15,6 +15,36 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
++
+{product-title} is designed for FIPS. When running {op-system-base-full} or {op-system-first} booted in FIPS mode, {product-title} core components use the {op-system-base} cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 Validation on only the x86_64, ppc64le, and s390x architectures.
```
Comment (GroceryBoyJr) [inline security/compliance_operator/compliance-operator-release-notes.adoc:39]:
```suggestion
{product-title} is designed for FIPS. When running {op-system-base-full} or {op-system-first} running in FIPS mode, {product-title} core components use the {op-system-base} cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 validation on only the x86_64, ppc64le, and s390x architectures.
```

Comment (GroceryBoyJr) [reply security/compliance_operator/compliance-operator-release-notes.adoc:39]: I suggested lowercase "v" and  "running" vs. "booted". I don't have an argument against "booted", I just thought "running" was more accurate - your call sir.

**Review by GroceryBoyJr — COMMENTED** [review]
Only a couple tiny nits sir. Otherwise LGTM!

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 3]
```diff
@@ -15,6 +15,36 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
+// ====
+// It is recommended to update the Compliance Operator to version 1.3.1 or later before updating your {product-title} cluster to version 4.14 or later.
+// ====
+
+[id="compliance-operator-1-3-1-new-features-and-enhancements"]
+=== New features and enhancements
+
+* You can install and use the Compliance Operator in an {product-title} cluster in FIPS mode.
++
+{product-title} is designed for FIPS. When running {op-system-base-full} or {op-system-first} running in FIPS mode, {product-title} core components use the {op-system-base} cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 validation on only the x86_64, ppc64le, and s390x architectures.
```
Comment (xiaojiey) [inline security/compliance_operator/compliance-operator-release-notes.adoc:39]: For the FIPS paragrapgh blow, it is proper to put it here? It is more related to FIPS itself not the Compliance Operator, seen from https://docs.openshift.com/container-platform/4.14/installing/installing-fips.html.

Comment (sheriff-rh) [reply security/compliance_operator/compliance-operator-release-notes.adoc:39]: I'll remove it and revert to your previously approved state. :+1:

**Comment by xiaojiey** [comment]
The preview doesn't work. But the PR content looks good to me.

**Comment by sheriff-rh** [comment]
The preview was still uploading. Current revision: https://file.rdu.redhat.com/antaylor/CO-1.3.1/security/compliance_operator/compliance-operator-release-notes.html#compliance-operator-release-notes-1-3-1

**Comment by sheriff-rh** [comment]
Content is live; merging.

**Comment by sheriff-rh** [comment]
git cherrypick enterprise-4.15

**Pull request merged**